### DISCUSSION TITLE
how to get cell from list and table

### DISCUSSION THREAD

**Discussion by Silentdoer** [nushell/nushell#7394] [Ideas]
such as:
[1,2]|select 0|value 0
will prints 1, it is a int value,not list<int>

ls|select 0|select name|value "0,name"
will prints filename, it is a string value, not table<name: string>

**Comment by Silentdoer** [comment]
value maybe use cell?

**Comment by Silentdoer** [comment]
oh, [1,2].0 can support

**Comment by fdncred** [comment]
It may be helpful to read through our documentation found here https://www.nushell.sh/
